rep-2001.rst

```diff
@@ -33,13 +33,16 @@ Specification
 End-user entry points
 ---------------------
 
-We define three main entry points for ROS users.
-
- * desktop (recommended): The `desktop` metapackage is the main entry point for users.
-   It provides all commonly used libraries as well as visualization tools and tutorials.
- * ros_base: The `ros_base` metapackage composes the `ros_core` metapackage with commonly used libraries.
+We define four main entry points for ROS users.
+
+ * `desktop_full` (recommended from Humble): The `desktop_full` metapackage is the main entry point for users.
```

Comment:
My preference is not to have a `desktop_full` variant at all.  I have two main reasons for this:

1. The name `desktop_full` is really meaningless; it gives no indication of what is in there, or what is expected to be in there.
2. Because it is so ambiguous, it means that it can be continually argued over what should and shouldn't be in it.

Reply:
I hear you. Here are some other suggestions:

1. quickstart
2. entrypoint
3. toolkit

Reply:
I agree the name isn't optimal. We could do `kitchen_sink`?? just kidding. We are putting it all in there. We used to have a automagic `all` metapackage. Which actually depended on all built packages, but that was deprecated a long time ago as being truely meaningless as to what you actually got as it was best effort. And you could just replace it with `ros-XXX-*` in apt.

I like the concept of `quickstart` where it's indicating that this is a shorthand guess at what you might want but isn't necessarily the end all of where you should end up for an application. We want to capture the idea of it being a default starting point but you don't have to stay with it once you know better. Looking at the [ubuntu metapackages](https://help.ubuntu.com/community/MetaPackages) they have a concept of `standard` Which by name is nice, but I think its closer to `ros-base` semantically if we want to keep parallels. 

`entrypoint` is a nice idea, but already is heavily overloaded with other meanings as technical keyword. 

`toolkit` has a similar problem with `desktop_full` as it doesn't give  a sense of scope.

So in the end, I'd suggest `quickstart` is best so far.

Reply:
Changed to `quickstart` in 86241d49b523304ce7d95f877f2e7cb38a5ace70.

Reply:
After a discussion with the ROS team, we'll go back to `desktop_full` for backwards compatibility with ROS 1: [c0dd46b](https://github.com/ros-infrastructure/rep/pull/334/commits/c0dd46bba6e7e24c949ee72a9e3174ee167bd958)